include/crm/services_internal.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2010-2021 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef __PCMK_SERVICES_INTERNAL__
+#  define __PCMK_SERVICES_INTERNAL__
```

Comment:
Technically the C standard reserves anything starting with double underbar, which is why we went with PCMK__